documentation/API_documentation/CFS-API-Readiness-Checklist.md

```diff
@@ -1,20 +1,21 @@
 
 # API Readiness Checklist
 
-Checklist for CFS API wip
+Checklist for API call-forwarding-signal v0.2.0-rc.1 in r1.1
+
 
 | Nr | API release assets  | alpha | release-candidate |  public-release<br>initial | public-release<br> stable | Status | Comments |
 |----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
 |  1 | API definition                               |   M   |         M         |    M    |    M   |   Y  | /code/API_definitions/Call_Forwarding_Signal.yaml |
 |  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |   Y  |      |
 |  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |   Y  |      |
-|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |   Y  |      |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |   Y  | v0.2.0-rc.1 |
 |  5 | API documentation                            |   M   |         M         |    M    |    M   |   Y  | inline in YAML |
-|  6 | User stories                                 |   O   |         O         |    O    |    M   |   Y  |      |
-|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |   N  | /documentation/API_documentation/CFS_UserStory_UseCase-1.md |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |   Y  | /documentation/API_documentation/CFS_UserStory_UseCase-1.md |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |   N  | code/Test_definitions/call-forwarding-signal-every-forwarding.feature, code/Test_definitions/call-forwarding-signal-unconditional.feature |

```

Comment:
```suggestion
|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |   N  | /code/Test_definitions/call-forwarding-signal-every-forwarding.feature, /code/Test_definitions/call-forwarding-signal-unconditional.feature |
```

Also these links are relative to the repository root (like the others).

BTW: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links